src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/helpers.js

```diff
@@ -92,10 +92,27 @@ function showValidation() {
     : displayValidationErrors();
 }
 
+function createShowConfirmationForm() {
+  if (document.querySelector('#showConfirmationForm')) {
+    return;
+  }
+  const template = document.createElement('template');
+  // eslint-disable-next-line no-undef
+  const form = `<form method="post" id="showConfirmationForm" name="showConfirmationForm" action="${ShowConfirmationPaymentFromComponent}">
```

Comment:
How does this variable get filled? Can we pass it as a parameter instead of having it just pop up here?

Reply:
you mean `${ShowConfirmationPaymentFromComponent}`?
I tried passing `window.ShowConfirmation` but it didn't work